docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1096 @@
+
+            /// Unset the callback
+            VoiceControlManagerClient.PreRecognitionResultUpdated -= preRecognitionResultUpdated;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the all recognition result received callback invoked when voice control engine sends the all recognition result to voice control manager.
+
+    In the callback, the recognized result, recognized text, and engine message results are included and recognized result can include more than two voice commands if two voice commands those have same command string are registered by each voice control clients.
+
+    If you want to select specific command in the recognized result, you can use SetRecognizedCommandsSelectionDelegate() method.
+
+    ```csharp
+    /// callback
+    void allRecognitionResultReceived(object sender, AllRecognitionResultEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetAllRecognitionResultReceivedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.AllRecognitionResultReceived += allRecognitionResultReceived;
+
+            /// Unset the callback
+            VoiceControlManagerClient.AllRecognitionResultReceived -= allRecognitionResultReceived;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the recognition result updated callback invoked when voice control engine set the recognition result updated to voice control manager.
+
+    You can get the recognized result, recognized text, and recognized command list in this callback.
+
+    If no commands are matched, the callback returns `RecognizedResult.Rejected`:
+
+    ```csharp
+    /// callback
+    void recognitionResultUpdated(object sender, RecognitionResultUpdatedEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetRecognitionResultUpdatedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.RecognitionResultUpdated += recognitionResultUpdated;
+
+            /// Unset the callback
+            VoiceControlManagerClient.RecognitionResultUpdated -= recognitionResultUpdated;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the recognized commands selection callback invoked when the voice control manager select specific voice commands in all recognized result:
+
+    When utterence is recognized, all recognized result can include more than two voice commands if more than two voice commands those have same command string are registered by each voice control clients.
+
+    In this case, voice control manager can select specific voice commands from all recognized result to send voice control client using SelectRecognizedCommandsDelegate.
```

Comment:
@sungraejo shouldn't we write `SelectRecognizedCommandsDelegate` as inline code block. Please let me know if it doesn't.